docs/docs/guide/05-scavenge/08-cli.md

```diff
@@ -71,16 +72,20 @@ func CmdCommitSolution() *cobra.Command {
 			solutionHashString := hex.EncodeToString(solutionHash[:])
 
 			// convert a scavenger address to string
-			var scavenger = clientCtx.GetFromAddress().String()
+			scavenger := clientCtx.GetFromAddress().String()
 
 			// find the hash of solution and scavenger address
-			var solutionScavengerHash = sha256.Sum256([]byte(solution + scavenger))
+			solutionScavengerHash := sha256.Sum256([]byte(solution + scavenger))
 
 			// convert the hash to string
-			var solutionScavengerHashString = hex.EncodeToString(solutionScavengerHash[:])
+			solutionScavengerHashString := hex.EncodeToString(solutionScavengerHash[:])
 
 			// create a new message
-			msg := types.NewMsgCommitSolution(clientCtx.GetFromAddress().String(), solutionHashString, solutionScavengerHashString)
+			msg := types.NewMsgCommitSolution(
+        clientCtx.GetFromAddress().String(),
+        solutionHashString,
+        solutionScavengerHashString
```

Comment:
```suggestion
        solutionScavengerHashString,
```

Once this is fixed, you can run a tool that I created to format go code inside md snippet.

```
$ go run github.com/tbruyelle/mdgofmt/cmd/mdgofmt@latest -w docs/docs/guide/05-scavenge/*.md
```

After that all the go snippets are well formatted with the standard gofmt. You can eventually use it in others folders. Note that if the code doesn't compile (like if you run it w/o the suggestion above), it will return an error.

Reply:
Formatting the code by hand bored me ;)